Models/articles-model.js

```diff
@@ -1,5 +1,19 @@
 const db = require('../db/connection');
 
+exports.selectArticles = () => {
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
+      FROM articles
+      LEFT JOIN comments ON articles.article_id = comments.article_id
+      GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url
+      ORDER BY created_at DESC;`
+    )
```

Comment:
Really nice work with your SQL query and using the COUNT aggregate function. 
As an extra stretch you could look in to using an alias for your table names but this is food for thought rather than something that needs changing now.